tests/e2e/interOp/pytest.ini

```diff
@@ -0,0 +1,50 @@
+[pytest]
```

Comment:
@shivamcandela  should this information

Reply:
We need to remove this file and use the main pytest.ini

Reply:
fixed/updated